scripts/statistics.js

```diff
@@ -6,6 +6,7 @@
 const chalk = require('chalk');
 
 const bcd = require('..');
+const version = require('../package.json').version;
```

Comment:
We can skip a require with `process.env.npm_package_version`:

```suggestion
```

Reply:
Ooh, I didn't know about that, I'll start using that in my other projects too!